pages/utilities/image-resizer.tsx

```diff
@@ -0,0 +1,284 @@
+        width,
+      });
+    }
+  }, [imageFile, width, height, format, quality, maintainAspectRatio]);
+
+  const qualityInput = useMemo(() => {
+    if (format === "jpeg") {
+      return (
+        <div className="flex flex-col flex-1">
+          <Label>Quality (0.1 to 1.0)</Label>
+          <Input
+            type="number"
+            min="0.1"
+            max="1.0"
+            step="0.1"
+            value={quality}
+            onChange={(e) => setQuality(parseFloat(e.target.value))}
+            disabled={!imageFile}
+            className="h-[32px] rounded-md"
+          />
+        </div>
+      );
+    }
+
+    return null;
+  }, [format, imageFile, quality]);
+
+  return (
+    <main>
+      <Meta
+        title="Image Resizer by Jam.dev | Free, Open Source & Ad-free"
+        description="Resize images online with Jam's free and open source Image Resizer. Adjust dimensions, maintain aspect ratio, and choose between PNG and JPEG formats."
+      />
+      <Header />
+      <CMDK />
+
+      <section className="container max-w-2xl mb-12">
+        <PageHeader
+          title="Image Resizer"
+          description="Resize images while maintaining aspect ratio and choose between PNG and JPEG formats."
+        />
+      </section>
+
+      <section className="container max-w-2xl mb-6">
+        <Card className="flex flex-col p-6 hover:shadow-none shadow-none rounded-xl">
+          <div>
+            <ImageUploadComponent onFileSelect={handleFileSelect} />
+
+            <div className="flex justify-between items-center mb-2 pt-6">
+              <div className="flex-1 mr-2">
+                <Label className="mb-2">Width (px)</Label>
+                <Input
+                  type="number"
+                  placeholder="Enter width"
+                  onChange={handleWidthChange}
+                  value={width ?? ""}
+                  className="mb-2"
+                  disabled={!imageFile}
+                />
+              </div>
+              <div className="flex-1 ml-2">
+                <Label className="mb-2">Height (px)</Label>
+                <Input
+                  typeof="number"
+                  placeholder="Enter height"
+                  onChange={handleHeightChange}
+                  value={height ?? ""}
+                  className="mb-2"
+                  disabled={!imageFile}
+                />
+              </div>
+            </div>
+
+            <div className="flex items-center mb-6 gap-2">
+              <Checkbox
+                id="mantain-aspect-ratio"
+                checked={maintainAspectRatio}
+                onCheckedChange={handleAspectRatioChange}
+                disabled={!imageFile}
+                className="mr-1"
+              />
+              <Label
+                htmlFor="mantain-aspect-ratio"
+                className="mb-0 hover:cursor-pointer"
+              >
+                Maintain Aspect Ratio
```

Comment:
I feel like wording should be `Preserve Aspect Ratio`. It sounds more descriptive to end user